features/borrowers_view_and_sign_deed.feature

```diff
@@ -18,255 +18,67 @@ Scenario: Single borrower views and signs deed, then views
   And the borrower <1> signature element is present on page
   Then the deed is digitally signed by borrower <1>
   And borrower <1> views the deed
+  And the mortgage deed is displayed
   Then I verify borrower <1> has signed the deed
 
 @us133 @us133f
-Scenario: Single borrower views deed, then views and signs
-  Given I create default deed with <1> borrowers
-  And borrower <1> views the deed
-  And borrower <1> views the deed
-  Then the deed is digitally signed by borrower <1>
-
-@us133 @us133f
-Scenario: Single borrower views deed 3 times
-  Given I create default deed with <1> borrowers
-  And borrower <1> views the deed
-  And borrower <1> views the deed
-  Then borrower <1> views the deed
```

Comment:
The step "Then the deed is digitally signed by borrower <1>" performs the signing process.

A borrower viewing the deed seemed a bit redundant, I'm happy to put it back in.